framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
I'm reading the arguments on both sides of this. I have mixed reactions:

- I agree that it should not be possible, with a Command, to affect a disabled widget in any way (with one exception). A Command for "Paste" must not paste into a text field if that text field is disabled. The exception is that setting up a command to enable or disable all of the widgets to which it is attached seems perfectly reasonable.
- It must remain possible for a single command to be attached to multiple buttons. That's how menus and other buttons that execute the same feature work in general, and is a long-time feature of qooxdoo. Breaking that would likely break many, many programs. It's documented and does exactly what it's supposed to.

So I'd propose that the bug that a disabled widget can be affected (other than its enabled state) by a Command be fixed. I'd further propose (and feel very strongly) that nothing be done that would prevent a single Command from being attached to multiple widgets.